pyproject.toml

```diff
@@ -0,0 +1,43 @@
+[build-system]
+requires = ["setuptools>=61.2", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "spdx-tools"
+authors = [{name = "Ahmed H. Ismail", email = "ahm3d.hisham@gmail.com"}]
+maintainers = [
+    {name = "Philippe Ombredanne", email = "pombredanne@gmail.com"},
+    {name = "SPDX group at the Linux Foundation and others"},
+]
+license = {text = "Apache-2.0"}
+description = "SPDX parser and tools."
+readme = "README.md"
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.6",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+urls = {Homepage = "https://github.com/spdx/tools-python"}
+requires-python = ">=3.6"
+dependencies = ["ply", "rdflib", "click", "pyyaml", "xmltodict"]
+dynamic = ["version"]
+
+[project.scripts]
+pyspdxtools_convertor = "spdx.cli_tools.convertor:main"
```

Comment:
this is going to rename the scripts, which means that the docs need updating. at least the readme https://github.com/spdx/tools-python#command-line-usage

And if we're renaming these, which IMO is a good idea as these names are too generic, can we please change "convertor" to "converter" in order to be using the main word in the dictionaries. c.f. https://thecontentauthority.com/blog/convertor-vs-converter

Reply:
See https://trends.google.com.au/trends/explore?geo=AU&q=convertor,converter for more evidence that convertor is not common.
And look up the word in dictionaries.

Reply:
v0.7.0 will be dropping support for Python 2, which is already a breaking change.

And `parser` & `convertor` were added after v0.6.1, so they havent been released yet.  Now is the time to change them.

https://pypi.org/project/spdx-tools/0.6.1/ was released August 2019, before https://github.com/spdx/tools-python/pull/148

Reply:
Thanks for pointing it out! I had only seen the date of the Github release (which was created July 2021, two years after the tag and pypi release were created...) and made assumptions without actually checking. Let's rename then.